api-samples/declarativeNetRequest/no-cookies-manager/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "no-cookies Rule Manager",
+  "version": "0.1",
+  "manifest_version": 3,
+  "description": "Manage in which conditions cookies are denied to be send",
+  "background": {
+    "service_worker": "service_worker.js"
+  },
+  "action": {
+    "default_title": "no-cookies Rule Manager",
+    "default_popup": "popup.html"
+  },
+  "host_permissions": ["<all_urls>"],
+  "permissions": [
+    "declarativeNetRequest",
+    "declarativeNetRequestFeedback",
+    "activeTab"
+  ],
```

Comment:
@oliverdunk can you double-check that all these permissions are needed?

Reply:
I checked again and found that the `activeTab` permission is not necessary. I will correct it right away.